recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,133 @@
+from conans import CMake, ConanFile, tools
+import os
+import string
+
+
+class AwsSdkCppConan(ConanFile):
+    name = "aws-sdk-cpp"
+    description = "The AWS SDK for C++ provides a modern C++ (version C++ 11 or later) interface for Amazon Web Services (AWS)."
+    topics = ("conan", "aws", "amazon", "cloud", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aws/aws-sdk-cpp"
+    license = "Apache-2.0",
+    exports_sources = "CMakeLists.txt",
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    short_paths = True
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_rtti": [True, False],
+        "use_virtual_operations": [True, False],
+        "with_curl": [True, False],
+        "enable_curl_logging": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_rtti": True,
+        "use_virtual_operations": True,
+        "with_curl": True,
+        "enable_curl_logging": False,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 2003)
```

Comment:
Why 2003?

Reply:
I'm wondering myself.
https://github.com/aws/aws-sdk-cpp/blob/2dffafc32c419bc6be45d6345fef0cd50eec3bd3/CMakeLists.txt#L73 says current is 11.

Reply:
Keep 11. 2003 it's not listed as default cppstd in setting.yml.

Reply:
I'll make the change to 11.
But 2003 is a fine value too.
Last time I checked, `tools.check_min_cppstd` converts `self.settings.compiler.cppstd` and `11` to a 4-digit year,
That way comparing `98` and `11` uses the same code path.

Reply:
Btw, using 98,11,14,17 will cause a new y2k-kind problem in less then 80 years. 🤣